evm_arithmetization/src/fixed_recursive_verifier.rs

```diff
@@ -1330,6 +1226,11 @@ where
     /// - `generation_inputs`: a transaction and auxiliary data needed to
     ///   generate a proof, provided
     /// in Intermediary Representation.
+    /// - max_cpu_len_log: logarithm of the max number of CPU cycles in the
+    ///   current segment.
+    /// - segment_index: index of the segment to prove (i.e. segment 0 proves
+    ///   the first `max_cpu_len` cycles, the second proves the following
```

Comment:
```suggestion
    ///   the first `max_cpu_len` cycles, segment 1 proves the following
```
Just to prevent any risk of ambiguity.